zerver/openapi/curl_param_value_generators.py

```diff
@@ -63,6 +66,18 @@ def iago_message_id() -> Dict[str, int]:
         "message_id": helpers.send_stream_message(helpers.example_email("iago"), "Denmark")
     }
 
+@openapi_param_value_generator(["/messages/{message_id}/reactions:delete"])
+def add_emoji_to_message() -> Dict[str, List[Dict[None, None]]]:
+    api_key = helpers.example_user('iago').api_key
+    client = cl(
+        email='iago@zulip.com',
+        api_key=api_key,
+        site='http://zulip.zulipdev.com:9981'
+    )
+    add_reaction(client, 41)
```

Comment:
Where does the `41` come from?

Reply:
I think it'd also be better to use `helpers` rather than the API bindings, for consistency with other similar code.